docs/repositories-configure/local-analysis/client-side-tools.md

```diff
@@ -1,12 +1,14 @@
 # Client-side tools
 
-Client-side tools enable you to run analysis either locally or as part of your CI pipeline, and then integrating the results into Codacy. This way, Codacy presents the analysis information reported by your local tools together with all other code quality information on the Codacy dashboards.
+Client-side tools enable you to run an analysis locally or as part of your CI pipeline and upload the results to Codacy. This way, Codacy presents the analysis information reported by your local tools and all other code quality information on the Codacy dashboards.
 
 Codacy supports client-side tools in two ways:
 
--   **Containerized tools:** Codacy provides a Docker image for the tools, and you run the images using the [Codacy Analysis CLI](running-local-analysis.md).
+-   **Containerized tools:** Codacy provides Docker images to run analysis tools locally. To run the images, print out analysis results, optionally upload them to Codacy, and more, use the [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli#codacy-analysis-cli) (command-line interface).
 
-    The Codacy Analysis CLI automatically fetches the code pattern settings that you define on the Codacy UI and applies them when running these tools.
+    The Codacy Analysis CLI lets Codacy present on its dashboards the analysis information reported by any tools you may be using, in addition to the default code quality information, and is necessary to upload analysis data for tools that analyze locally compiled assets.
+
+    If an [API token](../../codacy-api/api-tokens.md) is set, the Codacy Analysis CLI automatically fetches the code pattern settings that you define on the Codacy UI and applies them when running the tools.
```

Comment:
I think these paragraphs are trying to achieve too much! :slightly_smiling_face: 

- It seems like we're "promoting" the Codacy Analysis CLI ("To run the images, print out analysis results, optionally upload them to Codacy, and more [...]"), but we can simplify and stick to the core use case of:

    1. Fetching the code pattern configurations from Codacy
    2. Running the client-side tool Docker images "locally"
    3. Uploading the results back to Codacy.

    Adding to this, we don't need to take into account all possible flags for running the Codacy Analysis CLI. For example, we can remove "If an API token is set" because in the instructions for running the client-side tools we'll be telling the users to set it up.

- The advantages mentioned in the paragraph "The Codacy Analysis CLI lets Codacy present on its dashboards the analysis information reported by any tools you may be using [...] actually also apply to the standalone tools. :slightly_smiling_face: So, it seems weird to only mention them for the Codacy Analysis CLI. What we can do instead is complement the intro of the page with these details, since it applies to both the containerized and the standalone client-side tools.